src/components/CHeader.vue

```diff
@@ -2,11 +2,43 @@
   <header class="shadow w-screen">
     <!-- Icon made by <a href="https://www.flaticon.com/authors/eucalyp" title="Eucalyp">Eucalyp</a> from <a href="https://www.flaticon.com/" title="Flaticon"> www.flaticon.com</a> -->
     <nav>
-      <nav class="flex items-center justify-between flex-wrap bg-wiremind p-6">
+      <nav class="flex items-center justify-between flex-wrap bg-wiremind p-6 fixed w-full">
         <div class="flex items-center flex-shrink-0 text-white mr-6">
-          <img src="../assets/img/hero-white.png" width="34" height="34" class="mr-3" alt="" />
+          <img
+            src="../assets/img/hero-white.png"
+            width="34rem"
+            height="34rem"
+            class="mr-3"
+            alt=""
+          />
           <span class="font-mono font-semibold text-xl tracking-tight ">Bowl</span>
         </div>
+        <div class="flex">
+          <div>
+            <a
+              @click="onUpdateMessages"
+              href="#"
+              class="inline-block bg-gray-300 mr-1 rounded text-sm px-1 py-1 leading-none   text-white  hover:border-transparent hover:text-teal-500 hover:bg-white"
+            >
+              <img src="@/assets/img/update.png" width="20rem" height="20rem" />
```

Comment:
It's better to use svg image when possible in a web app (way smaller and no scale problems). You can take this one for example: https://material.io/resources/icons/?search=refresh&icon=refresh&style=baseline.

Reply:
there is a small course for svg icons with tailwind if you want https://tailwindcss.com/course/working-with-svg-icons/

Reply:
Or follow https://google.github.io/material-design-icons/ icons for web web to have a lot of icons via a font.